### DISCUSSION TITLE
Which networks support walletconnect

### DISCUSSION THREAD

**Discussion by moncokarl** [WalletConnect/walletconnect-monorepo#1641] [Developer Support]
hi, sry if choose wrong thread. 
afrter explore of website, I don't undersatnd do u support blockcahain, or do u accept only Etherium and Solana networks?

**Comment by chadyj** [comment]
WalletConnect v2 is chain-agnostic, and can work with many blockchains.

Have a look at our sample apps with support for multiple chains such as Solana, Polkadot, and more.
Note: WalletConnect supports many more chains. There are just a few highlighted in the examples for illustration purposes.

https://react-app.walletconnect.com/
https://react-wallet.walletconnect.com/

**Reply by moncokarl** [reply]
thx for answer.
One more question, in supported apps I see BNB, but at your exmple I dont see blockchain network, do u support blockhain networks?

**Reply by kuchta** [reply]
@moncokarl What do you mean by blockchain network? Basically all networks are blockchain networks....

**Reply by moncokarl** [reply]
Hi is it possible to connect and get payments via walletconnect tokens on TRC20, BEP20, and coins - BTC, BCH, XRP etc.?
